src/lazy/value.rs

```diff
@@ -66,9 +66,17 @@ impl<'top, D: Decoder> LazyValue<'top, D> {
         LazyValue { expanded_value }
     }
 
-    fn symbol_table(&'top self) -> &'top SymbolTable {
+    #[cfg(feature = "experimental-tooling-apis")]
+    pub fn symbol_table(&'top self) -> &'top SymbolTable {
```

Comment:
🗺️ When the `experimental-tooling-access` feature is enabled, values can allow access to their symbol table.